### PR TITLE
Upgrade QuicTrace to .NET 5

### PR DESCRIPTION
**Pull request by ThadHouse**
.NET 5 has been out for a while, and most tools should be updated. Also gives us some nice performance boosts.

### CODE REVIEW

File: src/plugins/trace/exe/Program.cs [hunk 1]
```diff
@@ -234,7 +234,7 @@ static void Main(string[] args)
                     Console.Write("quictrace> ");
 
                     var input = Console.ReadLine();
-                    if (input == "--exit" || input == "exit" || input == "-e")
+                    if (input == null || input == "--exit" || input == "exit" || input == "-e")
```
Comment (nibanks) [inline src/plugins/trace/exe/Program.cs:237]: Does `null` mean "User hit enter" or "User hit ctrl+c"?

Comment (ThadHouse) [reply src/plugins/trace/exe/Program.cs:237]: Null means end of file. So if user passes in a file through stdin null will be returned after the last line. Always been able to be return, but 5.0 properly annotated Console.ReadLine.

Comment (nibanks) [reply src/plugins/trace/exe/Program.cs:237]: Ok. I just want to make sure "User hit enter" doesn't exit.

Comment (ThadHouse) [reply src/plugins/trace/exe/Program.cs:237]: Nope. That will return an empty string.

**Review by nibanks — APPROVED** [review]

**Pull request merged**